guide/src/advanced/troubleshooting/cross-comp-config.md

```diff
@@ -279,3 +279,21 @@ that it needs in order to relay on behalf of a consumer chain.
 
 ### Fix
 Set `ccv_consumer_chain = true` in `config.toml`.
+
+## Mishandling Clock Drift
```

Comment:
This seems to be covered already in https://github.com/informalsystems/hermes/blob/master/guide/src/advanced/troubleshooting/cross-comp-config.md#maximum-block-time